mm2src/coins/utxo.rs

```diff
@@ -734,6 +742,39 @@ impl UtxoAddressScanner {
     }
 }
 
+/// Contains lists of mature and immature UTXOs.
+#[derive(Debug, Default)]
+pub struct MatureUnspentList {
+    mature: Vec<UnspentInfo>,
+    immature: Vec<UnspentInfo>,
+}
+
+impl MatureUnspentList {
+    pub fn with_capacity(capacity: usize) -> MatureUnspentList {
```

Comment:
Yeah, thanks for noticing, I try to find all places where `inline` attribute should be used 👍

Reply:
Done. I hope I put `inline(always)` in the right places. Please check it

Reply:
We should be careful with using `inline(always)`, I think `inline` should be enough https://gist.github.com/kvark/f067ba974446f7c5ce5bd544fe370186#dont-use-inlinealways what do you think @ozkanonur?

Reply:
> We should be careful with using `inline(always)`, I think `inline` should be enough https://gist.github.com/kvark/f067ba974446f7c5ce5bd544fe370186#dont-use-inlinealways what do you think @ozkanonur?

I prefer to use just `#[inline]` almost all the time. But I think you can prefer using `always` anyway. That documentation is absolutely true if you are inlining a function that does multiple jobs rather than a simple, single-purpose small function. For example, if you have a function that deals with multiple tasks and that function called from 500 different part of the code, if you use `inline` even without `always` that  wouldn't be a good idea because that will basically bloat assembly insturictions and binary output.

However, for functions like this:
```rs
pub fn some_fn(x: &SomeInstanceType) -> YType {
     x.get_something("blabla").unwrap();
}

.
.
.

pub fn main_flow() {
    .
    .
    .
    let y = some_fn(&x);
    .
    .
    .
}
```

You can use `inline(always)` without any problem. Since having `let y = x.get_something("blabla").unwrap();` instead of `let y = some_fn(&x);` wouldn't bloat anything at all. Inlining is a little bit tricky stuff, shouldn't be used everywhere but should definetly be used for cases like similar to example above.

Reply:
I agree with @shamardy, thank you all for the advices. I replace `#[inline(always)]` with `#[inline]` almost everywhere.